style/layout.css

```diff
@@ -0,0 +1 @@
+/*Still not exactly sure what to put into the layout css file...*/
```

Comment:
It can definitely be tricky figuring out what goes in layout versus base versus modules. Here's a good article that I find useful for determining the differences. https://smacss.com/book/type-layout.
 In general, the layout.css file should target larger sections such as #header, #article, #footer, and modules.css should target smaller sections within layout.css.